build.bash

```diff
@@ -8,48 +8,48 @@
 #
 # Outputs compiled program in current directory.
 # Default file name is 'ecaddy'.
-#
+
 set -e
 
-output="$1"
-if [ -z "$output" ]; then
-	output="ecaddy"
-fi
+: ${output_filename:="$1"}
+: ${output_filename:="ecaddy"}
 
 pkg=main
+ldflags=()
 
 # Timestamp of build
-builddate_id=$pkg.buildDate
-builddate=`date -u`
+name="${pkg}.buildDate"
+value=$(date --utc +"%F %H:%M:%SZ")
+ldflags+=("-X" "\"${name}=${value}\"")
```

Comment:
You trolling me here, right? I cannot tell.

There has been no old format. It showed different for every *locale.*

I'd really like not to be the one to re-introduce any particular local format. Follow with a new commit and be verbose about order and rendering of all fields?

----

```sh
$ docker run -ti --rm blitznote/debootstrap-amd64:16.04
# date -u

Sat 2016-03-19 09:51:17 +0000

# vanilla glibc, "C"

Sat Mar 19 09:54:36 UTC 2016

# vanilla glibc, "zh_CN.UTF-8"

2016年 03月 19日 星期六 10:15:56 UTC
```